contracts/sfc/SFCBase.sol

```diff
@@ -42,7 +42,7 @@ contract SFCBase is SFCState {
     error WrongValidatorStatus();
 
     // requests
-    error RequestedCompleted();
+    error AlreadyRedirected();
```

Comment:
I'd put it below `SameRedirectionAuthorizer` to the same group, since it is not used for requests in general.